src/apprt/gtk-ng/class/window.zig

```diff
@@ -101,6 +101,40 @@ pub const Window = extern struct {
             );
         };
 
+        pub const @"titlebar-style-native" = struct {
+            pub const name = "titlebar-style-native";
+            const impl = gobject.ext.defineProperty(
+                name,
+                Self,
+                bool,
+                .{
+                    .nick = "Titlebar style is native",
+                    .blurb = "True if the titlebar style is native",
+                    .default = true,
+                    .accessor = gobject.ext.typedAccessor(Self, bool, .{
+                        .getter = Self.getTitlebarStyleNative,
+                    }),
+                },
+            );
+        };
+
+        pub const @"titlebar-style-tabs" = struct {
+            pub const name = "titlebar-style-tabs";
+            const impl = gobject.ext.defineProperty(
+                name,
+                Self,
+                bool,
+                .{
+                    .nick = "Titlebar style is tabs",
+                    .blurb = "True if the titlebar style is tabs",
+                    .default = true,
+                    .accessor = gobject.ext.typedAccessor(Self, bool, .{
+                        .getter = Self.getTitlebarStyleTabs,
+                    }),
+                },
+            );
```

Comment:
Can't we use a GEnum here? That's what I would do in C. I'm not sure what zig-gobject pattern in though.

Reply:
This is mainly for the Blueprint, but I think it makes a lot more sense if the Blueprint calls a closure instead (`bind template.is-titlebar-style-tabs()`)

Reply:
If using a blueprint bind leads us to writing less than optimal code (my opinion), I would not use the blueprint bind. I don't think we are saving any lines of code with this.